pkg/models/live.go

```diff
@@ -5,12 +5,16 @@ import (
 	"encoding/json"
 	"time"
 
+	"github.com/centrifugal/centrifuge"
```

Comment:
We got rid of `centrifuge` recently in models to not depend on it in parts other than `live` package.

Reply:
I'll change the function to just get the count -- that is all I am using now